### PR TITLE
Refactor/rewrite user interface

### PR DESCRIPTION
**Pull request by Derstilon**
image
image

### CODE REVIEW

File: src/ThreeEditor/ThreeEditor.tsx [hunk 1]
```diff
@@ -36,18 +53,77 @@ function ThreeEditor(props: ThreeEditorProps) {
...
+							theme.palette.mode === 'dark'
+								? theme.palette.secondary.dark
+								: theme.palette.secondary.light
+					}
+				}}>
+				<Typography
+					sx={{
+						padding: 2
+					}}>
+					Tutaj będzie panel properites
```
Comment (grzanka) [inline src/ThreeEditor/ThreeEditor.tsx:117]:
```suggestion
					Placeholder for the properties panel
```

File: src/ThreeEditor/ThreeEditor.tsx [hunk 2]
```diff
@@ -36,18 +53,77 @@ function ThreeEditor(props: ThreeEditorProps) {
...
+					<Button
+						onClick={() => {
+							document.getElementById('sidebar')?.classList.toggle('hidden');
+						}}>
+						Ukryj strare UI
```
Comment (grzanka) [inline src/ThreeEditor/ThreeEditor.tsx:122]:
```suggestion
						Hide old UI
```

File: src/ThreeEditor/main.js [hunk 3]
```diff
@@ -27,16 +27,16 @@ export function initEditor(container) {
...
 	const toolbar = new Toolbar(editor);
-	container.appendChild(toolbar.dom);
+	// container.appendChild(toolbar.dom);
 
 	const sidebar = new Sidebar(editor);
 	container.appendChild(sidebar.dom);
 
 	const menubar = new Menubar(editor);
-	container.appendChild(menubar.dom);
+	// container.appendChild(menubar.dom);
```
Comment (grzanka) [inline src/ThreeEditor/main.js:36]: why is this commented out ?

Comment (Derstilon) [reply src/ThreeEditor/main.js:36]: This UI was being replaced and now i can delete it.

**Review by grzanka — DISMISSED** [review]
LGTM

File: src/ThreeEditor/ThreeEditor.tsx (hunk 1 above)
Comment (grzanka) [inline src/ThreeEditor/ThreeEditor.tsx:117]: Why have you reverted this polish stuff ?

Comment (Derstilon) [reply src/ThreeEditor/ThreeEditor.tsx:117]: I had to revert some changes because they broke Viewport but somehow it reverted translated strings as well

**Comment by ostatni5** [comment]
image
The colors of the text and the background are not the best match.

**Review by ostatni5 — COMMENTED** [review]
image
The colors of the text and the background are not the best match.

**Comment by grzanka** [comment]
> image The colors of the text and the background are not the best match.

I agree with @ostatni5

**Comment by Derstilon** [comment]
@ostatni5 @grzanka 
image

image

**Comment by Derstilon** [comment]
I could also add this background
image

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**